app/bundles/EmailBundle/Controller/EmailController.php

```diff
@@ -369,8 +369,11 @@ public function viewAction($objectId)
                 new \DateTime($dateRangeForm->get('date_from')->getData()),
                 new \DateTime($dateRangeForm->get('date_to')->getData())
             );
+            $statsDevices = $model->getEmailDeviceStats($email, $variant, new \DateTime($dateRangeForm->get('date_from')->getData()),
+                new \DateTime($dateRangeForm->get('date_to')->getData()));
```

Comment:
Although I didn't find this in the code standards, I personally find it a bit hard to read. Maybe the following code would be more organized. What do you think?
```
$statsDevices = $model->getEmailDeviceStats(
    $email,
    $variant,
    new \DateTime($dateRangeForm->get('date_from')->getData()),
    new \DateTime($dateRangeForm->get('date_to')->getData())
);
```